python/tank/util/shotgun/download.py

```diff
@@ -156,9 +155,63 @@ def download_and_unpack_attachment(sg, attachment_id, target, retries=5, auto_de
         the bundle in a subfolder, this should be correctly unfolded.
     :raises: ShotgunAttachmentDownloadError on failure
     """
+    return _download_and_unpack(
+        sg,
+        target,
+        retries,
+        auto_detect_bundle,
+        attachment_id=attachment_id
+    )
+
+
+def download_and_unpack_url(sg, url, target, retries=5, auto_detect_bundle=False):
```

Comment:
please double check that this new method made it into the sphinx docs. (I tried to search for it and couldn't find it, but maybe the search didn't pick it up correctly for local setups?)

Reply:
It was not -- in fact the old method wasn't either.  I added them under a new heading in the `Utilities` section.  [tk-core.zip](https://github.com/shotgunsoftware/tk-core/files/3039062/tk-core.zip)